test/components/Pickers.test.jsx

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
+import { mount } from 'enzyme';
 import 'raf/polyfill';
 import Pickers from '../../src/components/Pickers';
 
 it('renders correctly', () => {
-  const tree = renderer
-    .create((
-      <Pickers
-        platform="win10"
-        benchmark="overview"
-        onChange={() => {}}
-      />
-    ))
-    .toJSON();
-  expect(tree).toMatchSnapshot();
+  const wrapper = mount(
+    <Pickers platform="win10" benchmark="overview" onChange={() => {}} />,
+  );
+  expect(wrapper).toMatchSnapshot();
```

Comment:
Enzyme changes are looking good! Thanks!

Reply:
The snapshots are ugly IMO. See https://github.com/mozilla-frontend-infra/firefox-performance-dashboard/pull/226#discussion_r333010488